api/src/utils/index.js

```diff
@@ -97,6 +97,59 @@ const updatePlacesCenter = async (center) => {
   return center;
 };
 
+const updateCenterDependencies = async (center) => {
+  try {
+    const youngs = await YoungModel.find({ cohesionCenterId: center._id });
+    youngs.forEach(async (young) => {
+      young.set({
+        cohesionCenterName: center.name,
+        cohesionCenterZip: center.zip,
+        cohesionCenterCity: center.city,
+      });
+      await young.save();
+    });
+    const referents = await ReferentModel.find({ cohesionCenterId: center._id });
+    referents.forEach(async (referent) => {
+      referent.set({ cohesionCenterName: center.name });
+      await referent.save();
+    });
+    const meetingPoints = await MeetingPointModel.find({ centerId: center._id });
+    meetingPoints.forEach(async (meetingPoint) => {
+      meetingPoint.set({ centerCode: center.code });
+      await meetingPoint.save();
+    });
+  } catch (e) {
+    console.log(e);
+  }
+};
+
+const deleteCenterDependencies = async (center) => {
+  try {
+    const youngs = await YoungModel.find({ cohesionCenterId: center._id });
+    youngs.forEach(async (young) => {
+      young.set({
+        cohesionCenterId: null,
+        cohesionCenterName: null,
+        cohesionCenterZip: null,
+        cohesionCenterCity: null,
```

Comment:
D'après ce que je lis ici : https://stackoverflow.com/questions/4486926/delete-a-key-from-a-mongodb-document-using-mongoose

```suggestion
        cohesionCenterId: undefined,
        cohesionCenterName: undefined,
        cohesionCenterZip: undefined,
        cohesionCenterCity: undefined,
```